charts/ndb-operator/templates/apiextensions.k8s.io_v1_customresourcedefinition_databases.ndb.nutanix.com.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
crd need to be put in a dedicated directory like explained here
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you